apis/bigtable/v1beta1/logicalview_types.go

```diff
@@ -40,13 +53,20 @@ type BigtableLogicalViewStatus struct {
 	// A unique specifier for the BigtableLogicalView resource in GCP.
 	ExternalRef *string `json:"externalRef,omitempty"`
 
+	// The unique name of the BigtableLogicalView. Values are of the form
+	//  `projects/{project}/instances/{instance}/logicalViews/{logicalViewID}`.
+	// +kcc:proto:field=google.bigtable.admin.v2.LogicalView.name
+	Name *string `json:"name,omitempty"`
+
 	// ObservedState is the state of the resource as most recently observed in GCP.
 	ObservedState *BigtableLogicalViewObservedState `json:"observedState,omitempty"`
 }
 
 // BigtableLogicalViewObservedState is the state of the BigtableLogicalView resource as most recently observed in GCP.
 // +kcc:proto=google.bigtable.admin.v2.LogicalView
 type BigtableLogicalViewObservedState struct {
+	// The observed state of the LogicalView object.
+	Spec BigtableLogicalViewSpec `json:"spec,omitempty"`
```

Comment:
ObservedState should not include a field if it is only supposed to be configured by user.